tests/udf/udf_server.py

```diff
@@ -13,9 +13,11 @@
 # limitations under the License.
 
 import datetime
-from udf import *
 from decimal import Decimal
 import time
+from typing import List, Dict, Any, Tuple, Optional
```

Comment:
Add a comments how to install `databend_udf` package?

Reply:
This is just a test file but not document, more details would be in https://github.com/databendcloud/databend-udf